frontend/src/components/filters/FiltersInfoTS.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { FILTER_TYPE } from "../../constants";
+import { FilterDescription, FilterSet, FilterValue, RangeFilterValue, StringArrayFilterValue, isMetadataFilterValue, isRangeFilterValue, isStringArrayFilterValue } from "../../models/paged_items";
```

Comment:
There are a couple of unused imports. You can use the VS Code command "Organize Imports" to clean out any unused imports in your source file (cmd-shift-P opens the command palette).